src/mechanics/integrands/Base.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <memory>
+
+namespace NuTo
+{
+namespace Integrand
```

Comment:
Do we need this namespace? I'd rather have `NuTo::Integrand` as the base class. Also, the namespace is called `Integrand`, and the directory `integrands`.

Reply:
Hmm. In the directory, you have many integrands. I just saw that the laws in /constitutive/laws are also in namespace `Laws` instead of `Law`. I think it's reasonable to have a singular word in the namespace. If I'm the only one, I'll change it!

Anyways, I vote for keeping a namespace. Maybe, that looks a bit strange for the base class. The alternative is to have the name `Integrand` in each class name. /mechanics/integrands/IntegrandTimeDependent.*/ (Or do you have another suggestion?).  I'd rather use `Integrand::TimeDependent`.